extensions/vi-mode/visual.lisp

```diff
@@ -248,3 +248,28 @@
         (move-to-column *start-point* end-col)
         (move-to-column (current-point) start-col))
       (vi-visual-swap-points)))
+
+(defmethod lem/language-mode:set-region-point ((start point) (end point) (global-mode (eql :|vi|)))
+  (declare (ignore global-mode))
+  (when (visual-p)
+    (let ((v-range (visual-range)))
+      (move-point start (car v-range))
+      (move-point end (cadr v-range)))))
+
+
+(eval-when (:compile-toplevel :load-toplevel)
+  (sb-ext:unlock-package :lem-base)
```

Comment:
Don't we want a SBCL feature flag? Or see https://quickdocs.org/cl-package-locks which defines lock/unlock for 4 implementations.